src/MediaWiki/Connection/TransactionHandler.php

```
@@ -15,17 +15,11 @@
 
-	/**
-	 * @var string|null
-	 */
-	private $sectionTransaction;
+	private ?string $sectionTransaction = null;
 
-	/**
-	 * @var bool|null
-	 */
-	private $mutedTransactionProfiler;
+	private ?bool $mutedTransactionProfiler = null;
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$mutedTransactionProfiler'. (undefined)

(UnusedPrivateField)